apps/core/src/server.ts

```diff
@@ -17,13 +24,21 @@ async function start() {
 
   app.withTypeProvider<ZodTypeProvider>();
   await app.listen({ port: Config.PORT, host: Config.HOST });
-
+  // start queue right after the app
+  await import('./queue/setup.js');
```

Comment:
After this line, you should add `await addSyncToQueue({ operation: '', disciplinaModel: DisciplinaModel });` so the syncMatriculas queue starts running every 2 minutes.